frontend/src/styles/App.scss

```diff
@@ -23,12 +23,20 @@ body {
   left: 0;
   right: 0;
   top: 0;
-  background: rgba(0, 0, 0, 0.5);
   transition: 0.5s ease;
-  opacity: 0;
   font-size: 20px;
-  padding: 20px;
+}
+
+.overlay-background {
+  background: rgba(0, 0, 0, 0.5);
```

Comment:
Define a fallback background above. Eg:
```
background: #000000; /* This is the fallback color, for browsers that don't support rgba */
background: rgba(0, 0, 0, 0.5); /* This will overwrite and be used as the background in supporting browsers */
```